diff -Naur a/meson.build b/meson.build
--- a/meson.build	2025-02-21 23:43:00.000000000 -0500
+++ b/meson.build	2025-02-22 22:42:46.528099457 -0500
@@ -1900,7 +1900,7 @@
 
 dep_clang = null_dep
 if with_clc or with_gallium_clover
-  llvm_libdir = dep_llvm.get_variable(cmake : 'LLVM_LIBRARY_DIR', configtool: 'libdir')
+  llvm_libdir = get_option('clang-libdir')
 
   dep_clang = cpp.find_library('clang-cpp', dirs : llvm_libdir, required : false)
 
diff -Naur a/meson.options b/meson.options
--- a/meson.options	2025-02-21 23:43:00.000000000 -0500
+++ b/meson.options	2025-02-22 22:43:58.344461585 -0500
@@ -763,6 +763,13 @@
 )
 
 option(
+  'clang-libdir',
+  type : 'string',
+  value : '',
+  description : 'Locations to search for clang libraries.'
+)
+
+option(
   'mesa-clc',
   type : 'combo',
   value : 'auto',
